monai/networks/nets/hovernet.py

```diff
@@ -423,6 +433,7 @@ class HoVerNet(nn.Module):
             repository, each user is responsible for checking the content of model/datasets and the applicable licenses
             and determining if suitable for the intended use. please check the following link for more details:
             https://github.com/vqdang/hover_net#data-format
+        freeze_encoder: whether to freeze the encoder of the network.
```

Comment:
"whether to freeze the encoder of the network when executing the backpropagation."
What do you think?

Thanks.

Reply:
Hi @Nic-Ma , IMO, we may not need to bind "freeze" with "executing the backpropagation". I did not see this kind of description in other places.

Reply:
OK, I see.